### ISSUE TITLE
🐛 [Bug] Southshore Guard & Tarren Mill Deathguard incorrect progression level

### ISSUE THREAD

**Issue by Daribon** [vmangos/core#1730]
## 🐛 Bug report

Southshore Guard & Tarren Mill Deathguard are currently level 55 no matter what server patch you got set.

### Expected behavior

Southshore Guard & Tarren Mill Deathguard should be level 50 below patch 1.7.0.

Tarren Mill Deathguard level 50, unknown date:
![inside taurenmill](https://user-images.githubusercontent.com/6137576/206770335-e2b1e059-79b4-446d-9913-28df2d47dd26.jpg)

Tarren Mill Deathguard level 50, unknown date:
![tauren mill](https://user-images.githubusercontent.com/6137576/206770494-f0aa3eb2-1a2f-4a8d-9bbf-e0a15f8dd480.jpg)

Tarren Mill Deathguard level 55, unknown date:
![134 - FNSWY](https://user-images.githubusercontent.com/6137576/206772618-c35e60a7-36f0-482a-a296-57d8e1a383d1.jpg)

Tarren Mill Deathguard level 50 in tooltip, 2005 unknown date:
https://youtu.be/A6Y-6ESCNuE?t=148

Tarren Mill Deathguard level 50, dated 2005/05/31:
![wow_screen_20050604_040144_3882_00073](https://user-images.githubusercontent.com/6137576/206770850-80c63a24-d77e-4972-a5e2-1f0e6524988e.jpg)

Tarren Mill Deathguard level 50, dated 2005/06/07:
![wow_screen_20050607_170030_4108_00007](https://user-images.githubusercontent.com/6137576/206771031-d6a7d37b-4769-41bf-839d-a4fe9ea2d938.jpg)

Southshore guard level 50, unknown date:
![td8](https://user-images.githubusercontent.com/6137576/206770579-1fbccdee-e550-4a3b-b384-be761cfa7fd2.jpg)

Southshore Guard level 50, unknown date:
![557 - ykjhy](https://user-images.githubusercontent.com/6137576/206773049-c0b94f66-ef10-4ec8-ad24-db4b038b148d.png)

Southshore Guard level 50, dated 2005/03/26:
![WoWScrnShot_032605_030253](https://user-images.githubusercontent.com/6137576/206772182-a05a261a-62ad-4fa9-b8b6-73da5000df25.png)

Southshore guard level 50 from video dated 2005:
https://www.youtube.com/watch?v=5hc4Ap50Zrg&t=100s

Southshore Guard level 55, dated 2005/12/13:
![wow_screen_20051215_175521_9023_00052](https://user-images.githubusercontent.com/6137576/206771789-b9543499-67bb-4dbb-ad38-96496bad78f6.jpg)

Tarren Mill Deathguard level 50, 2004/11/14, before release:
https://web.archive.org/web/20041114165905/http://wow.allakhazam.com/db/mob.html?wmob=2405

Allakhazam lists both Southshore Guard and Tarren Mill Deathguard as level 50 from January 2005 to April 2005:  
https://web.archive.org/web/20050413211841/http://wow.allakhazam.com:80/dyn/mobs/zone26.html

September 8, Allakhazam showing Southshore guards and Tarren Mill Deathguard suddenly starts getting listed as level 45-50.
https://web.archive.org/web/20050413211841/http://wow.allakhazam.com/dyn/mobs/zone26.html

September 13, Allakhazam showing levevel 45-50:
https://web.archive.org/web/20050913055534/http://wow.allakhazam.com:80/dyn/mobs/zone26.html

September 29 Allakhazam showing as level 45-55:
https://web.archive.org/web/20050929235443/http://wow.allakhazam.com:80/dyn/mobs/zone26.html

Allakhazam continues to show level 45-55 until end of vanilla.

I have not found a single proof from in game that either Tarren Mill Deathguard or Southshore Guard was ever level 45.
I believe it was a bug on their website, considering is suddenly starts showing up in September 8.
Only proof of them being level 45 is from TBC, someone most likely uploaded data from tbc alpha to Allakhazam (TBC Friends & Family alpha began only a month prior to 1.7.0.)

As for the some changing level differences at some of the Allakhazam links, like level 49-50, or 47-50. I'm certain that's because of drunk effect screwing up data.

Considering they were listed as level 45 to level 50 in September 13 and then changed to level 45-55 in September 13 I believe the level change came in 1.7.0.

**TL;DR:
Southshore Guard & Tarren Mill Deathguard should be level 50 until patch 1.7.0.
They were changed to level 55 in patch 1.7.0.**

### Steps to reproduce

1. .tele southshore
2. Target Southshore Guard
3. Check level
4. .tele Tarren
5. Target Tarren Mill Deathguard
6. Check level

### Version & Environment

Client Version: 1.12.1.5875

Commit Hash:  https://github.com/vmangos/core/commit/4d7e82f4c14cb4360f21122ede6d3d1c0b7613cb

OS Client: Microsoft Windows 11 Pro N 10.0.22621.900
OS Server: Microsoft Windows 11 Pro N 10.0.22621.900

### Crashlog

- None

**Comment by ghost** [comment]
Accurate, Hillsbrad foothill town guards should be a mixture of levels; from 45-50.

**Comment by Daribon** [comment]
> Accurate, Hillsbrad foothill town guards should be a mixture of levels; from 45-50.

Please provide any in game proof of them ever being level 45.
I have not seen a single screenshot or video of them being level 45 at any point, release, beta or alpha.

Here is an image showing Southshore guard from first closed beta patch 0.5.5 (March 2004), before release:
![images_3826](https://user-images.githubusercontent.com/6137576/206782657-3ed84442-6fcd-4212-8659-0330fc85f250.jpg)

Here is the only image I could find showing Tarren Mill Deathguard being level 45:
![wowscrnshot080208085420](https://user-images.githubusercontent.com/6137576/206783301-978ddcc9-5bfc-46ad-8677-aa2eb10405fc.jpg)
It is from TBC, showing netherweave bag and all.

**Comment by ghost** [comment]
https://wowwiki-archive.fandom.com/wiki/Tarren_Mill_Deathguard
I am not sure how old this wiki is but they apparently have moved so I am sure its dated.
https://www.gamermaps.net/world-of-warcraft/npc/Tarren+Mill+Deathguard
These are all old links
https://www.gamermaps.net/world-of-warcraft/map/npc/Tarren+Mill+Deathguard
Another one
https://tbc.cavernoftime.com/npc=2405

I have not found any video of them being 45 either mainly because the videos are so blurry and dated using hypercams or cheap fraps.

but I have a fond recollection of them being a mixture of 45-50 at one point as i've been on Darkspear since 2005, Darkspear was known for world pvp and I spent a lot of time in leveling zones more than I wanted to because of it

As for the southshore guards they were always level 50 because they were more spread out in placement.

**Comment by Daribon** [comment]
>  https://tbc.cavernoftime.com/npc=2405

TBC private server database, invalid

> https://wowwiki-archive.fandom.com/wiki/Tarren_Mill_Deathguard

Wowwiki page created in 2008.

> https://wowwiki-archive.fandom.com/wiki/Tarren_Mill_Deathguard

Non vanilla database.

Please provide in game evidence, screenshot or recording.

**Comment by ghost** [comment]
You also have to consider that a good majority of TBC players played vanilla for many years, so the vanilla knowledge is still relevant.  Vanilla was only around for almost 3 years before TBC dropped

**Comment by ghost** [comment]
But I remember them being 45.

**Comment by Daribon** [comment]
> You also have to consider that a good majority of TBC players played vanilla for many years, so the vanilla knowledge is still relevant. Vanilla was only around for almost 3 years before TBC dropped

I'm asking for evidence, not reason to not provide evidence.

**Comment by ghost** [comment]
you are asking for evidence from a game where people coudln't even get decent screenshots or video recordings from that time, even the best pvper videos are at 180p or 240p

**Comment by ghost** [comment]
2006 was not a good time to be making recordings of ingame footage.

**Comment by Daribon** [comment]
> you are asking for evidence from a game where people coudln't even get decent screenshots or video recordings from that time, even the best pvper videos are at 180p or 240p

I have 23k sorted screenshots from vanilla, your argument is invalid.
https://github.com/Daribon/Vanilla-Screenshot-Archive

**Comment by ghost** [comment]
probably from late 2006

**Comment by Daribon** [comment]
Even a 180p or 240p video would be good enough.

People remember wrong all the time, which is why evidence is very important.
Guess work leads to inaccuracy and is therefore not accepted.

**Comment by ghost** [comment]
https://www.youtube.com/watch?v=xxlX6YM2ZUI
This is the only video i can find from 2004

**Comment by ghost** [comment]
But there is a whole section from 2019 on classic wow that has players talking about them being lvl 45/50, there was a patch that added more south shore guards along side the creek area, where you did the hecular quest.

**Comment by Daribon** [comment]
> https://www.youtube.com/watch?v=xxlX6YM2ZUI This is the only video i can find from 2004

The Tarren Mill Deathguards are level 50 in the video.
The video is from beta, before release and would be invalid even if they were lower than level 50.

> But there is a whole section from 2019 on classic wow that has players talking about them being lvl 45/50, there was a patch that added more south shore guards along side the creek area, where you did the hecular quest.

Provide evidence.

**Comment by ghost** [comment]
There is a severe lack luster of evidence in hillsbrad foothills, sorry I can't you're just going to have to trust that tarren mill used a mixture of guards from 45-50 years ago, south shore guards were always 50 due to the lack of them and how open south shore is as a town geographically

**Comment by Daribon** [comment]
> There is a severe lack luster of evidence in hillsbrad foothills, sorry I can't you're just going to have to trust that tarren mill used a mixture of guards from 45-50 years ago, south shore guards were always 50 due to the lack of them and how open south shore is a town geographically

That doesn't make any sense, why would allakhazam list both Southshore Guard and Tarren Mill Deathguard as level 45-50 then only between September 8 and September 13. Later on Allakhazam shows them as level 45-55 for both Southshore Guard and Tarren Mill Deathguard.

**Comment by ghost** [comment]
Don't know, but I remember seeing 45 level 48 and level 50 TM Deathguards. 

And southshore guards were always 50 and there was a severe lack of them before later patches. 

I distinctively remember running towards the highest level TM guard when allies would chase me back to town

This is the only explanation of the guards being 55 "The new project lead wanted to kill-off spontaneous world-PvP ASAP. They wanted everyone raiding or in BGs, completely missing the point of the world that Mark Kern oversaw being created. That meant making both TM and SS virtually impenetrable.

Also, Hecular is de-activated or bugged. I finished the quest yesterday, corpse running repeatedly closer to his gravestone, so even if you can make it with a mad-rush with a whole group, Hecular won’t be there to save you and melt away the guards."

**Comment by Daribon** [comment]
> Don't know, but I remember seeing 45 level 48 and level 50 TM Deathguards.
> 
> And southshore guards were always 50 and there was a severe lack of them before later patches.

Any actual evidence to your claim?

> I distinctively remember running towards the highest level TM guard when allies would chase me back to town

What you remember has to be based on evidence. If I remember being able to teleport to Blasted Lands from Stormwind it doesn't mean it was like that in vanilla.

---

> But there is a whole section from 2019 on classic wow that has players talking about them being lvl 45/50, there was a patch that added more south shore guards along side the creek area, where you did the hecular quest.

Players you are referring to are remembering private servers and not actual vanilla.
They weren't spawned previously in mangos, because mangos database is based on ancient databases which had bad sniffing. 
Southshore guards were actually spawned in the creek area in entire vanilla.

To prove my point, here is an image from wowemu patch 1.4.0 (emulator from 2005), which mangos most likely originally based the spawns on:
![WoW_0H3Hb1wHGy](https://user-images.githubusercontent.com/6137576/206792598-9df8020d-ea31-4054-8905-2c3ca6b77f23.jpg)
Entire area empty, no turtles or guards, due to bad sniffing

While the rest southshore being spawned in wowemu 1.4.0:
![WoW_hTFCNhaCiN](https://user-images.githubusercontent.com/6137576/206792284-c9778cc4-fd5a-4aa1-9556-2f63cb7c1b4d.jpg)

**Comment by ghost** [comment]
They were there by the creek but there wasn't as many that there are now

**Comment by Daribon** [comment]
> They were there by the creek but there wasn't as many that there are now

I know exactly the reason why.
To dig even deeper, let's check SDB from 2006. SDB later became mangos database.
Keep in mind that SDB was horribly spawned, guess work everywhere.
![WoW_HXWsuADcGa](https://user-images.githubusercontent.com/6137576/206799401-27cb73bb-263b-4ff1-a231-e2759eef3084.jpg)
Just one guard spawn!

All the guards are level 55 in SDB.
In the image I posted from wowemu previously, all guards were level 50.

People are remembering private servers, not vanilla.
Can you start providing evidence now?

**Comment by ghost** [comment]
you can think that the guards were level 55 in vanilla all you want, but my memory hasn't served me wrong yet in life, they were not level 55 until before TBC released and no one world pvped there any longer because of so, I distinctly remember.

**Comment by Daribon** [comment]
> you can think that the guards were level 55 in vanilla all you want, but my memory hasn't served me wrong yet in life, they were not level 55 until before TBC released and no one world pvped there any longer because of so, I distinctly remember.

???
Your reply makes no sense whatsoever, did you even read through the bug report.
I've proved several times, both with videos and images that they should be level 55 in later vanilla, but level 50 in early vanilla.

**Comment by ghost** [comment]
I don't care anymore make them whatever level you want

**Comment by Daribon** [comment]
Great.
